### PR TITLE
VERIFY-XDP-MTU: New Testcase to verify various MTU Sizes

### PR DESCRIPTION
**Pull request by abhimarathe**
This Testcase will test XDP working with various MTU sizes (1500, 2000, 3506) which are easily configurable in XML for future maintenance. Also, it will verify error caught by kernel "hv_netvsc" for MTU greater than Maximum MTU on Azure.

```
[LISAv2 Test Results Summary]
Test Run On           : 06/05/2020 08:23:58
ARM Image Under Test  : Canonical : UbuntuServer : 18.04-LTS : latest
Initial Kernel Version: 5.3.0-1022-azure
Final Kernel Version  : 5.7.0-rc5-next-20200514
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:38

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes)
-------------------------------------------------------------------------------------------------------------------------------------------
    1 XDP                  VERIFY-XDP-MTU                                                                    PASS                 7.82


Logs can be found at C:\LISAv2\WG98\TestResults\2020-06-05-01-23-52-5061
```

### CODE REVIEW

File: XML/TestCases/FunctionalTests-XDP.xml [hunk 1]
```diff
@@ -109,4 +109,25 @@
...
+            <param>mtuSizes=(1500 2000 3506)</param>
+            <param>defaultMtu=1500</param>
+            <param>maxMtuSize=3506</param>
+            <param>repo_url=XDPDUMP_REPO_URL</param>
+        </TestParameters>
+        <Platform>Azure</Platform>
+        <Category>Functional</Category>
+        <Area>XDP</Area>
+        <Tags>xdp,network,hv_netvsc,sriov</Tags>
+        <Priority>1</Priority>
```
Comment (juhlee-microsoft) [inline XML/TestCases/FunctionalTests-XDP.xml]: Should be 3 until this kernel is under distro pickup schedule?

File: Testscripts/Windows/VERIFY-XDP-MTU.ps1 [hunk 2]
```diff
@@ -0,0 +1,138 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+
+<#
+.Description
+    This script deploys the VM, verify XDP working with various MTU sizes (1500, 2000, 3506) which are easily configurable in XML.
```
Comment (juhlee-microsoft) [inline Testscripts/Windows/VERIFY-XDP-MTU.ps1]: VM and verified XDP

File: Testscripts/Windows/VERIFY-XDP-MTU.ps1 [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+.Description
+    This script deploys the VM, verify XDP working with various MTU sizes (1500, 2000, 3506) which are easily configurable in XML.
+    Also, it will verify error caught by kernel "hv_netvsc" for MTU greater than Maximum MTU on Azure.
+#>
+
+param([object] $AllVmData,
+    [object] $CurrentTestData)
+
+$MIN_KERNEL_VERSION = "5.6"
+$iFaceName = "eth1"
```
Comment (juhlee-microsoft) [inline Testscripts/Windows/VERIFY-XDP-MTU.ps1:14]: Can you read this interface name from system?

**Review by juhlee-microsoft — CHANGES_REQUESTED** [review]

File: Testscripts/Windows/VERIFY-XDP-MTU.ps1 (hunk 3 above)
Comment (abhimarathe) [reply Testscripts/Windows/VERIFY-XDP-MTU.ps1:14]: This is a synthetic NIC name. This is going to be same in Azure environment.

**Review by juhlee-microsoft — APPROVED** [review]

**Pull request merged**